dev-tools/template.yml

```diff
@@ -0,0 +1,62 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Developer tool infrastructure
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  Environment:
+    Description: The environment type
+    Type: String
+    AllowedValues:
+      - dev
+      - build
+      - staging
+      - integration
+      - production
+
+Resources:
+  EmptyS3BucketsFunction:
+    #checkov:skip=CKV_AWS_116:DLQ not required as failures will be reported to CloudFormation
+    #checkov:skip=CKV_AWS_117:VPC not required
+    Condition: DeployS3Cleanup
```

Comment:
Need to remove this condition as it no longer applies now it's deployed separately